src/SurveyService.Models.Dto/Requests/Question/CreateQuestionRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Option;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question;
+
+public record CreateQuestionRequest
+{
+  public Guid? GroupId { get; set; }
```

Comment:
If we want to create a group with 10 questions and we want all this questions to be in one group. We want to send only 1 request. Untill we create a group, we have no idea about its Id, we can't pass anything here.
In another scenario, if we want to add question to an existing group we can pass groupId, but most likely we will do it an query parameter. All in all, the idea is to separate groupId from the request.

Reply:
I propose to give GroupId an attribute [JsonIgnore], because if we send a tuple(CreateQuestionRequest, Guid?) to the validator, you still can't call it in CreateGroupRequestValidator

Reply:
done